contentcuration/contentcuration/frontend/channelEdit/views/CurrentTopicView.vue

```diff
@@ -299,10 +299,14 @@
             id: 'resourceDrawer',
             viewMode: viewModes.COMPACT,
           });
+          this.$nextTick(() => {
+            this.handleResourceDrawerResize();
+          });
         } else {
           this.removeViewModeOverride({
             id: 'resourceDrawer',
           });
+          this.handleResourceDrawerResize(0);
```

Comment:
Since we're hiding the resource panel, it will always be 0. The reason for having it on adding is to be able to get the width of the panel once it's available